### PR TITLE
Created a playbook to help with extension installation automation

### PR DESCRIPTION
**Pull request by JadinLuong**
Signed-off-by: Jadin Luong <Jadin.Luong@ibm.com>

Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [X] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [ ] Bugfix 
- [X] Feature 
- [ ] Other... Please describe:

#### Relevant issues

This PR was created for stage 3 of #1199.

#### Changes proposed in this PR

This PR introduces a new ansible playbook which will help with the automation of installing extensions onto a Zowe instance.

#### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: playbooks/roles/ext/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# ============================================================================
+# Install Extensions onto Zowe
+- name: Install Zowe Extension
+  import_role:
+    name: zos
+    tasks_from: run_script
+  vars:
+    script_chdir: "{{ zowe_root_dir }}/bin"
+    script_filename: ./zowe-install-component.sh
+    script_parameters: "-d \"{{ zowe_extension_dir }}\" -i \"{{ zowe_instance_dir }}\" -o \"{{ work_dir_remote }}/{{ filename_to_upload }}\" -l \"{{ zowe_logs_dir }}\""
```
Comment (jackjia-ibm) [inline playbooks/roles/ext/tasks/main.yml:31]: Think `zowe_extension_filename` variable is better than `filename_to_upload`.

Comment (JadinLuong) [reply playbooks/roles/ext/tasks/main.yml:31]: You're right, it makes more sense.

**Comment by JadinLuong** [comment]
@jackjia-ibm, Made the changes as you requested above. I also realized there was a slight typo in the variable names causing some failures. Everything should be working as it should be now after the recent committed changes 👍.

File: playbooks/install-ext.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - import_role:
+      name: common
+  - import_role:
+      name: zos
+
+  # ============================================================================
+  # Check if instance is SMPE install, if so, we need to update zowe_root_dir
+  - import_role:
+      name: zos
+      tasks_from: detect_zowe_instance
```
[the remark this answers is not in the document]
Comment (jackjia-ibm) [reply playbooks/install-ext.yml:23]: Confirmed we do need this.

**Comment by JadinLuong** [comment]
Hey @jackjia-ibm, I've added the uninstall task for the extensions role. Here are the builds for the latest commit:

Convenience Build: https://wash.zowe.org:8443/view/Zowe/job/zowe-install-test/view/change-requests/job/PR-1918/15/
SMPe Build: https://wash.zowe.org:8443/view/Zowe/job/zowe-install-test/view/change-requests/job/PR-1918/16/
Docker Build: https://wash.zowe.org:8443/view/Zowe/job/zowe-install-test/view/change-requests/job/PR-1918/17/

**Review by jackjia-ibm — APPROVED** [review]
LGTM, thanks a lot Jadin for working on this.

**Pull request merged**